var/spack/repos/builtin/packages/libepoxy/package.py

```diff
@@ -29,6 +29,11 @@ class Libepoxy(AutotoolsPackage):
     """Epoxy is a library for handling OpenGL function pointer management for
     you."""
     homepage = "https://github.com/anholt/libepoxy"
-    url      = "https://github.com/anholt/libepoxy/releases/download/v1.3.1/libepoxy-1.3.1.tar.bz2"
+    url      = "https://github.com/anholt/libepoxy/releases/download/1.4.3/libepoxy-1.4.3.tar.xz"
+    list_url = "https://github.com/anholt/libepoxy/releases"
 
+    version('1.4.3', 'af4c3ce0fb1143bdc4e43f85695a9bed')
     version('1.3.1', '96f6620a9b005a503e7b44b0b528287d')
+
+    depends_on('meson')
+    depends_on('mesa')
```

Comment:
Would `depends_on('gl')` work here?

Reply:
There is no package called gl, but it's included in mesa.

Reply:
`gl` is a virtual package provided by `mesa` and `opengl`; the latter is supposed to be an external package that can be configured to use system-vendored OpenGL. The latter is supposed to perform better. See https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/opengl/package.py and #7063 for more details.

Reply:
It seems that the opengl in spack only provides gl@4.5, but here gl@9.2 is needed.

Reply:
9.2 is a mesa version. The `gl` package provides OpenGL spec versions.

Reply:
I tried to use depends_on('gl'), but it doesn't seem to work.